### ISSUE TITLE
Need to upgrade to new API version

### ISSUE THREAD

**Issue by bradoyler** [bradoyler/WePayASPNet#4]
Could use help with migration details. @vasusen @leblanc ?

**Comment by vasusen** [comment]
Hi Brad,

Sorry I did not see this earlier. I can help you with any questions or problems you have with the new version!

Vasusen

**Comment by ecnelson** [comment]
What new version are we talking about… Git or WePay?
I could offer some suggestions on the WePay side, but am an idiot when it comes to Git.
Eric

**Comment by bradoyler** [comment]
I'm guessing this is all I need to know https://www.wepay.com/developer/version/2014-01-08 
Although I noticed a http header 'api-version' needs to be added.

**Comment by radiusx2** [comment]
Upgrades need to be done by June 30.  Is anyone working on the update here?

**Comment by pcloadletter** [comment]
Hi - I was also wondering if anyone here is working on the required API update?  My company is using this SDK and we only have a week or two to upgrade.  Thanks in advance.

**Comment by mexner** [comment]
Did the upgrade happen?

**Comment by BrewDawg** [comment]
Yep, indeed this code no longer works, The "CheckoutCreateRequest" class has a ton of members in it that are rejected as errors now, such as "payer_email_message", "redirect_uri" and about 7 others, however, redirect_uri seems to be ingrained in the code. It worked for me up to this point, I'll see if I can fix it and post a version that works with their new changes
